app/src/main/java/com/connfa/ConnfaApplication.java

```diff
@@ -65,4 +68,13 @@ private void setupFabric() {
         );
         Fabric.with(this, new Crashlytics(), new TwitterCore(authConfig), new TweetUi());
     }
+
+    @MainThread
+    public ApplicationComponent applicationComponent() {
```

Comment:
components are accessed on the main thread, no need to do the double null checks and sheanigans